<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0.11828, y1:0.153409, x2:1, y2:1, stop:0 rgba(255, 68, 68, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>160</y>
      <width>150</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Helvetica&quot;;</string>
    </property>
    <property name="text">
     <string>Hello</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to your personal account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>560</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 18pt &quot;Helvetica&quot;;
background-color: rgb(255, 197, 102);</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>510</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 18pt &quot;Helvetica&quot;;
background-color: rgb(255, 197, 102);</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="character_image">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="skip_char">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>460</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 18pt &quot;Helvetica&quot;;
background-color: rgb(255, 197, 102);</string>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_char">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 18pt &quot;Helvetica&quot;;
background-color: rgb(255, 197, 102);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guess_char_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 18pt &quot;Helvetica&quot;;
background-color: rgb(255, 197, 102);</string>
    </property>
    <property name="text">
     <string>Guess</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="char_guess_text">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
